grafana-tikv-dashboard.md

```diff
@@ -5,236 +5,417 @@ category: reference
+- Blob prev duration：The time consumed when executing prev operations in Titan blob
+- Blob keys flow：The flow rate of operations on Titan blob keys
+- Blob bytes flow：The flow rate of bytes on Titan blob keys
+- Blob file read duration：The time consumed when reading Titan blob file
+- Blob file write duration：The time consumed when writing Titan blob file
+- Blob file sync operations：The count of blob file sync operations
+- Blob file sync duration：The time consumed when sync blob file
+- Blob GC action：The count of Titan GC actions
+- Blob GC duration：The Titan GC duration
+- Blob GC keys flow：The flow rate of keys read and written by Titan GC
+- Blob GC bytes flow：The flow rate of bytes read and written by Titan GC
+- Blob GC input file size：The size of Titan GC input file 
+- Blob GC output file size：The size of Titan GC output file
+- Blob GC file count：The count of blob files involved in Titan GC
+
+## Lock manager
+
+- Thread CPU：The CPU utilization of the lock manager thread
+- Handled tasks：The number of taks handled by lock manager
+- Waiter lifetime duration：The time consumed for the transaction waitting for the lock to be released
+- Wait table：The status information of wait table, including the number of locks and the number of transactions waitting for the lock
+- Deadlock detect duration：The time consumed for detecting deadlock
+- Detect error：The number of errors encountered when detecting deadlock, including the number of deadlocks
+- Deadlock detector leader：The information about the node where the deadlock detector leader is located
+
+## Memory
+
+- Allocator Stats：The statistics of the memory allocator
+
+## Backup
+
+- Backup CPU：The CPU utilization of the backup thread
+- Range Size：The histogram of backup range size
+- Backup Duration：The time consumed for backup
+- Backup Flow：The total bytes of backup
+- Disk Throughput：The disk throughput per instance
+- Backup Range Duration：The time consumed for range backup
+- Backup Errors：The number of errors encountered when making a backup
+
+## Encryption
+
+- Encryption data keys：The total number of encrypted data keys
+- Encrypted files：The number of encrypted files
+- Encryption initialized：It shows whether encryption is enabled, `1` means enabled.
+- Encryption meta files size：The size of meta file about encrpytion
+- Encrypt/decrypt data nanos：The histogram of time on encrypting/decrypting data ecch time
+- Read/write encryption meta duration：The time consumed for reading/writing encryption meta file
+
+## Explanation of Common Parameters
+
+### gRPC Message Type
+
+1. Transactional API：
+
+    - kv_get：The command of getting the latest version of data specified by ts
+    - kv_scan：The command of scanning a range of data
+    - kv_prewrite：The command of prewriting the data to be committed at first phase of 2PC
+    - kv_pessimistic_lock：The command of adding a pessimistic lock to the key to prevent other transaction from modifying
```

Comment:
```suggestion
    - kv_pessimistic_lock：The command of adding a pessimistic lock to the key to prevent other transaction from modifying this key
```